ide/microPG.ml

```diff
@@ -65,14 +65,21 @@ type 'c entry = {
 }
 
 let mC = [`CONTROL]
-let mM = [`MOD1]
+let mM () = if Coq_config.arch = "Darwin" then [`CONTROL;`MOD2;`META] else [`MOD1]
```

Comment:
I don't think you need a thunk here, since `arch` is a static value.

Reply:
Right. Thanks. Updated.